resources/test/sse_data_schema.json

```diff
@@ -1047,10 +1047,11 @@
           "type": "array",
           "items": {
             "$ref": "#/definitions/Operation"
-          }
+          },
+          "deprecated": true
```

Comment:
This addition to the schema has presumably come about due to the addition of the `deprecated` attribute, so seems like we'll need to remove that attribute in order to be strictly backwards compatible.  We'll need to handle this via updating the doc comments on the deprecated members/types only.